### PR TITLE
Modify configure/release script to work with pinned requirements.

### PR DESCRIPTION
**Pull request by Abhishek-Dev09**
Improve configure/release script to work of pinned requirements using github-pypi links.
Fixes : #2070 #2072 
Signed-off-by: Abhishek Kumar <abhishek.kasyap09@gmail.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -70,7 +70,7 @@ matrix:
       sudo: required
       script: 
         - source bin/activate
-        - ./bin/about check thirdparty/
+        - ./bin/about check thirdparty_general/
```
[the remark this answers is not in the document]
Comment (Abhishek-Dev09) [reply .travis.yml:73]: This directory contains old wheel that is to be removed

File: etc/conf/requirements_all.txt [hunk 2]
```diff
@@ -0,0 +1,244 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile --allow-unsafe --find-links=/var/folders/61/wbzyy9553cxfykg4_9vrjrcm0000gn/T/tmpyah53qw5 --generate-hashes --no-emit-find-links --output-file=requirements_all.txt --pip-args='--no-index'
```
[the remark this answers is not in the document]
Comment (Abhishek-Dev09) [reply etc/conf/requirements_all.txt:5]: pip compile use temp dir of host machine for generating req files

File: etc/scripts/deps_archive.py [hunk 3]
```diff
@@ -54,12 +53,6 @@ def generate_os_archive(links, requirement, archive_name):
             archive_name,
         ]
     run(pip_agrs)
-    root_dir = os.path.abspath(archive_name)
```
[the remark this answers is not in the document]
Comment (Abhishek-Dev09) [reply etc/scripts/deps_archive.py:57]: good catch!

File: requirements_all.txt [hunk 4]
```diff
@@ -0,0 +1,244 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
```
[the remark this answers is not in the document]
Comment (Abhishek-Dev09) [reply requirements_all.txt:3]: No, just for ur reference. 
Re Also I think we need a separate file for development requirements:
You mean to say for dev tools

File: tests/scancode/test_scancode_checks.py [hunk 5]
```diff
@@ -48,7 +48,7 @@
 @unittest.skipIf(not on_linux, 'Check about files only on one OS')
 class TestCheckAboutFiles(unittest.TestCase):
     def test_about_files_thirdparty(self):
-        subprocess.check_output('bin/about check thirdparty/'.split(), cwd=root_dir)
+        subprocess.check_output('bin/about check thirdparty_general/'.split(), cwd=root_dir)
```
[the remark this answers is not in the document]
Comment (Abhishek-Dev09) [reply tests/scancode/test_scancode_checks.py:51]: We have renamed thirdparty to thirdparty_general. Thirdparty contains only 2 files virtualenv.pyz

**Comment by pombredanne** [comment]
When I come to merge this, what would be the sequence I should use to effectively perform an upload of the wheels before they are deleted? (e.g. I can check out up to before the commit that deletes the wheels and then upload the wheels with which script and then continue?)

**Comment by Abhishek-Dev09** [comment]
> When I come to merge this, what would be the sequence I should use to effectively perform an upload of the wheels before they are deleted? (e.g. I can check out up to before the commit that deletes the wheels and then upload the wheels with which script and then continue?)

@pombredanne : merge after uploading wheels.

**Comment by pombredanne** [comment]
@Abhishek-Dev09 :wave: 
Can you confirm that this PR is no loger relevant nad has been broken down in two PRs: #2167 and #2184 ?

**Comment by Abhishek-Dev09-zz** [comment]
@pombredanne Yeah !  you are right , closing this !

**Comment by pombredanne** [comment]
@Abhishek-Dev09 Thanks! I am merging the other ones ATM

**Pull request closed without merging**